### PR TITLE
CB-3163 text overflow in header

### PR DESCRIPTION
**Pull request by tfokina**

### CODE REVIEW

File: webapp/packages/plugin-authentication-administration/src/Administration/IdentityProviders/AuthConfigurationsTable/AuthConfigurationsTable.tsx [hunk 1]
```diff
@@ -9,16 +9,20 @@
...
-  Table {
-    width: 100%;
-  }
-`;
+    Container {
+      overflow: hidden;
+    }
+    table-container {
+      overflow: auto;
+      max-height: calc(100vh - 152px); /* size for height overflow */
```
Comment (Wroud) [inline webapp/packages/plugin-authentication-administration/src/Administration/IdentityProviders/AuthConfigurationsTable/AuthConfigurationsTable.tsx]: we can't relay to constant size with `152px`, we need another approach

**Review by Wroud — CHANGES_REQUESTED** [review]

File: webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/TeamsTable/TeamsTable.tsx [hunk 2]
```diff
@@ -38,7 +32,7 @@ export const TeamsTable = observer<Props>(function TeamsTable({ teams, state, se
   const translate = useTranslate();
   const keys = teams.map(team => team.teamId);
 
-  return styled(useStyles(styles))(
+  return styled(useStyles(BASE_CONTAINERS_STYLES))(
```
Comment (Wroud) [inline webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/TeamsTable/TeamsTable.tsx]: does we need `BASE_CONTAINERS_STYLES`? we don't use container components here

File: webapp/packages/plugin-authentication-administration/src/Administration/IdentityProviders/AuthConfigurationsTable/AuthConfigurationsTable.tsx (hunk 1 above)
Comment (tfokina) [reply webapp/packages/plugin-authentication-administration/src/Administration/IdentityProviders/AuthConfigurationsTable/AuthConfigurationsTable.tsx]: Removed this

**Review by Wroud — APPROVED** [review]

File: webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/TeamsTable/TeamsTable.tsx (hunk 2 above)
Comment (tfokina) [reply webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/TeamsTable/TeamsTable.tsx]: Removed unnecessary styles

**Pull request merged**